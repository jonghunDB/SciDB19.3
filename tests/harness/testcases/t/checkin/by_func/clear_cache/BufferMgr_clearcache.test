--setup
--start-query-logging

--test
#
# test coverage for 3 rare blocks in CachedDBChunk.cpp
#

--start-igdata
apply(build(<v : int64> [I=0:0,1,0], I), clear_cache, test_clear_cache())

--stop-query-logging
