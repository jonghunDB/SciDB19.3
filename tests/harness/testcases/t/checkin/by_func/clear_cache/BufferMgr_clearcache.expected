SCIDB QUERY : <apply(build(<v : int64> [I=0:0,1,0], I), clear_cache, test_clear_cache())>
[Query was executed successfully, ignoring data output by this query.]

